ios/brave-ios/Sources/BraveVPN/Components/Subscription/BraveVPN+ReceiptResponse.swift

```diff
@@ -49,21 +62,21 @@ extension BraveVPN {
       return nil
```

Comment:
should we throw an error here too?

Reply:
I think no because of the comment there:

> // Receipt verification applies to Apple's IAP only,
> // if we detect Brave's SKU token we should not look at Apple's receipt.

So `nil` is valid, since this is not an error. It just means they got the receipt from Android or Desktop instead of from Apple's iAP. I'm not 100% sure, but it does seem to be working.